EvergreenUITests/EvergreenUITests.swift

```diff
@@ -0,0 +1,36 @@
+//
```

Comment:
It's there as a first step in the process of adding UI tests to the project. Just a EvergreenTests.swift is already in the project as a starting point / placeholder for unit tests.